src/SFA.DAS.ApprenticeCommitments.Web/Services/IBacklink.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Services
+{
+    public interface IBacklink

```

Comment:
Being a little picky here - objects that implement this interface have an *is-a* relationship to the interface.  The page models aren't really backlinks themselves though.  They *contain* back links.

A slightly more precise name would be `IContainsBackLink` or `IHasBackLink`, or maybe even `ICanGoBack`